### PR TITLE
Correct mfa info for Altra Federal Credit Union

### PR DESCRIPTION
**Pull request by afcuinfosec**
Altra supports sms and email based mfa

### CODE REVIEW

**Comment by mxxcon** [comment]
Is there a link to any documentation how to enable it?

**Comment by afcuinfosec** [comment]
> Is there a link to any documentation how to enable it?

Not that I can find, working to confirm.

**Comment by afcuinfosec** [comment]
I have confirmed that there is not any documentation at the moment

File: _data/banking.yml [hunk 1]
```diff
@@ -42,7 +42,9 @@ websites:
       twitter: AltraCU
```
Comment (mxxcon) [inline _data/banking.yml]:
```suggestion
```

File: _data/banking.yml [hunk 2]
```diff
@@ -42,7 +42,9 @@ websites:
...
       facebook: altra.org
```
Comment (mxxcon) [inline _data/banking.yml]:
```suggestion
```
twitter and facebook are not necessary if `tfa:yes`

**Review by mxxcon — COMMENTED** [review]
Looking at https://www.altra.org/digital/online-banking/online-banking-help, "Direct Deposit Information" section makes it sound like it's used for _authorization_, not _authentication_.

Can you confirm this?

**Comment by afcuinfosec** [comment]
> Looking at https://www.altra.org/digital/online-banking/online-banking-help, "Direct Deposit Information" section makes it sound like it's used for _authorization_, not _authentication_.
> 
> Can you confirm this?

I'm not sure exactly what that is talking about because I managed to lock myself out of my account so I can't log in to check, but yes 2fa may be configured for authentication - users are sent a otp during login if it's enabled.

Unrelated: sorry I'm new to github, is there something I need to do in order to get Travis to build correctly?

File: _data/banking.yml (hunk 2 above)
Comment (afcuinfosec) [reply _data/banking.yml]: Can I resolve this then?

**Comment by mxxcon** [comment]
> Unrelated: sorry I'm new to github, is there something I need to do in order to get Travis to build correctly?

Accept my 2 edit suggestions above, that will kick off travis with the new changes.
Or simply make those changes yourself.

**Review by mxxcon — APPROVED** [review]

**Review by stephengroat — APPROVED** [review]

**Pull request merged**